### PR TITLE
Readme zh anchor fix

### LINKED ISSUE #19621: Invalid Anchor in readme
**This is a Bug Report**
Invalid Anchor in readme 

**Proposed Solution:**
Update link
7a29e88009

**Page to Update:**
https://github.com/kubernetes/website/blob/master/README-zh.md

/assign
/language zh

### PR DESCRIPTION
**Pull request by Arhell**
#19621
/language zh

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The recommended way to run the Kubernetes website locally is to run a specialize
 <!--
 > If you'd prefer to run the website locally without Docker, see [Running the website locally using Hugo](#running-the-site-locally-using-hugo) below.
  -->
-> 如果您更喜欢在没有 Docker 的情况下在本地运行网站，请参阅下面的[使用 Hugo 在本地运行网站](#running-the-site-locally-using-hugo) 章节。
```
Comment (tanjunchen) [inline README-zh.md]: it is ok,but i prefer to `## 使用 Hugo 在本地运行网站 {#running-the-site-locally-using-hugo}`
/lgtm

Comment (tengqm) [reply README-zh.md]: No. The subsection now needs an explicit anchor.

Comment (tanjunchen) [reply README-zh.md]: > No. The subsection now needs an explicit anchor.

What do you mean ? 😄

Comment (tengqm) [reply README-zh.md]: I mean, line 125 in the file should be changed to 
`## 使用 Hugo 在本地运行网站 {#running-the-site-locally-using-hugo}`.

Comment (tanjunchen) [reply README-zh.md]: sure.
@Arhell 
thanks for your contribution.
could you update this?

Comment (Arhell) [reply README-zh.md]: I updated on the offer and received the output
使用 Hugo 在本地运行网站 {#running-the-site-locally-using-hugo} it title 
I think in markdown in the header it won’t work out that way

Comment (tengqm) [reply README-zh.md]: Try make the title a separate line? I mean add an empty line before and after the title line? We have been using this kind of custom anchors in many other places, for example: 

https://github.com/kubernetes/website/blob/master/content/zh/docs/concepts/storage/volume-snapshots.md

Comment (Arhell) [reply README-zh.md]: I understand, but as far as I know, readme is not displayed on the site, but we need the link to work.

Comment (tengqm) [reply README-zh.md]: AFAICT, proper rendering on the github is not a high priority today. Docs hosted here are further processed by many processors in the background and then rendered at k8s.io/docs/ site.

File: README-zh.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ The recommended way to run the Kubernetes website locally is to run a specialize
...
+> 如果您更喜欢在没有 Docker 的情况下在本地运行网站，请参阅下面的[使用 Hugo 在本地运行网站](#使用-hugo-在本地运行网站) 章节。
```
Comment (tengqm) [inline README-zh.md]: Change to line 97 should now be reverted for the link to work.

**Comment by Arhell** [comment]
@tengqm done but the link does not work 
https://github.com/Arhell/website/blob/zh-readme-fix/README-zh.md
What I wrote above, that the changes that I made were initially correct

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**